cadl-extension/pom.xml

```diff
@@ -86,6 +92,16 @@
       <build>
         <finalName>${project.artifactId}</finalName>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <includes>
+                <include>**/*Test.java</include>
+                <include>**/*Tests.java</include>
+              </includes>
+            </configuration>
```

Comment:
No idea why I need to configure it again.
This should already configured in parent POM, and works fine on other modules. Maybe different groupId?
https://github.com/Azure/autorest.java/blob/bfc0dfba9256128e2ca0de4bc5aabb991b149fbd/pom.xml#L101-L105